### DISCUSSION TITLE
Suggestion to follow single naming conventions for UWP project types.

### DISCUSSION THREAD

**Discussion by dishantmshah** [dotnet/maui#134] [General]
## Summary
As I was going through the project structure, I came across several projects which are related to UWP framework but the naming conventions used for those project folders/types are not consistent. 

## Current Naming Schemes

(In order of project structure)

1. [Windows Universal](https://github.com/dotnet/maui/tree/build/System.Maui.ControlGallery.WindowsUniversal)
2. [UWP](https://github.com/dotnet/maui/tree/build/System.Maui.Maps.UWP)
3. [UAP](https://github.com/dotnet/maui/tree/build/System.Maui.Platform.UAP)

## Intended Naming Scheme

A suggestion would be to use single naming convention for the given platform for better consistency. 

As per the current scenario when considering UWP development, UAP is the name used for DLL/Framework but branding is under UWP. So my personal opinion would be to use UWP in all the naming schemes as that's what most people are familiar with.

**Comment by mauler2025** [comment]
The current table on the readme page just said MICROSOFT, but what kind of MICROSOFT application is supported by MAUI? Are the application WPF, WinForm and UWP?
It is a bit confusing at a glance.
